pkg/helm/actions/auth.go

```diff
@@ -56,6 +57,24 @@ func setUpAuthenticationProject(chartPathOptions *action.ChartPathOptions, conne
 		chartPathOptions.KeyFile = tlsKeyFile.Name()
 		tlsFiles = append(tlsFiles, tlsKeyFile)
 	}
+	//set up basic auth
+	if connectionConfig.BasicAuthConfig != (configv1.SecretNameReference{}) {
+		secretName := connectionConfig.BasicAuthConfig.Name
+		secret, err := coreClient.Secrets(namespace).Get(context.TODO(), secretName, v1.GetOptions{})
+		if err != nil {
+			return nil, fmt.Errorf("failed to GET secret %q/%q, reason %v", secretNamespace, secretName, err)
```

Comment:
If you want to render the namespace/name with quotes used rather `%s` and wrapt the whole thing into quotes. Otherwise it will look a bit odd -> `'namespace'/'name'`
```suggestion
			return nil, fmt.Errorf("failed to GET secret '%s/%s', reason %v", secretNamespace, secretName, err)
```

Reply:
Here and below